omas/omas_physics.py

```diff
@@ -203,6 +203,47 @@ def core_profiles_zeff(ods, update=True, use_electrons_density=False):
                 prof1d_z['zeff'] = Z2n/Zn
     return ods_z
 
+
+def transform_current(rho, JtoR=None, JparB=None,
+                      equilibrium=None, includes_bootstrap=False):
+    """
+    Given <Jt/R> returns <J.B>, or vice versa
+    Transformation obeys <J.B> = (1/f)*(<B^2>/<1/R^2>)*(<Jt/R> + dp/dpsi*(1 - f^2*<1/R^2>/<B^2>))
```

Comment:
maybe point out in the doc string that both Jtor and JparB must be consistent with the equilibrium.cocosio